.github/workflows/ci.yml

```diff
@@ -61,14 +64,27 @@ jobs:
       - name: Build miscellanea
         run: make $coefficiency --output-sync=recurse -f install_miscellanea.make
 
+      - name: Cache libxml2, wxWidgets and wxPdfDoc
+        id: cache-local
+        uses: actions/cache@v2
+        with:
+          path: |
+            /opt/lmi/local/${{ env.LMI_COMPILER }}_${{ env.LMI_TRIPLET }}
+            /opt/lmi/local/include
+            /opt/lmi/local/share
+          key: build-${{ env.LMI_COMPILER }}-${{ env.gcc_version }}-${{ env.LMI_TRIPLET }}-${{ hashFiles('install_libxml2_libxslt.make', 'install_wx.sh', 'lmi\install_wxpdfdoc.sh') }}
```

Comment:
May be '*wx*.patch` and 'libx*.patch' or simply '*.patch' also should be in the list of hashed files.

Reply:
I don't think we use any patches for any of these libraries currently and I hope to switch to submodules later, so this isn't a problem.

The `lmi\` is an error though, both because it should use slash and not a backslash, and because it should be used with all files or none of them.